src/defines.h

```diff
@@ -26,214 +26,36 @@
 // ================================================
 
 // Set parameters of IMU and board used
+#ifndef IMU
 #define IMU IMU_AUTO
+#endif
+#ifndef SECOND_IMU
 #define SECOND_IMU IMU_AUTO
+#endif
+#ifndef BOARD
 #define BOARD BOARD_SLIMEVR_V1_2
+#endif
+#ifndef IMU_ROTATION
 #define IMU_ROTATION DEG_270
+#endif
+#ifndef SECOND_IMU_ROTATION
 #define SECOND_IMU_ROTATION DEG_270
+#endif
 
+#ifndef PRIMARY_IMU_OPTIONAL
 #define PRIMARY_IMU_OPTIONAL false
```

Comment:
Don't think wrapping these in ifndefs is necessary. Where else would the user define these? Honestly would just leave them there and if someone were to want to change these they can in place.

Reply:
they are needed for the build.py because it will redefine the board. 
I also think not the nicest solution. 

```ini
[env:BOARD_SLIMEVR_V1_2]
platform = espressif8266 @ 4.2.1
board = esp12e
build_flags =
  ${env.build_flags}
  -D BOARD=BOARD_SLIMEVR_V1_2
  ```
  
  Maybe nicer would be to have defines.h excluded if we run ci with -D NODEFINES=true ? And disable the include in globals.h? 
  So it would not affect the avarange diy person or webflasher.